h2o-py/tests/testdir_algos/xgboost/pyunit_H2OXGBoost_native_XGBoost_mixed_regression_compare_large_dense_NOPASS_multinode.py

```diff
@@ -34,9 +34,9 @@ def comparison_test_dense():
                    'gamma': 0.0,
                    'max_depth': h2oParamsD["max_depth"]}
 
-    nrows = random.randint(100000, 500000)
-    ncols = random.randint(1, 10)
-    factorL = random.randint(2, 10)
+    nrows = random.randint(10000, 100000)
```

Comment:
It is a good thing to speed up the tests a little bit ! Such quantity proved to be more than enough.